beacon_node/network/src/sync/block_lookups/tests.rs

```diff
@@ -155,7 +157,7 @@ impl TestRig {
     }
 
     #[track_caller]
-    fn expect_block_request(&mut self, response_type: ResponseType) -> Id {
+    fn expect_block_request(&mut self, response_type: ResponseType) -> SingleLookupReqId {
```

Comment:
Maybe rename this function now this could be either block and blob?

```suggestion
    fn expect_lookup_request(&mut self, response_type: ResponseType) -> SingleLookupReqId {
```

Reply:
Not part of this PR, but just below this under the `ResponseType::Blob` matach arm, the `SyncId` should be `SingleBlob` instead of `SingleBlock`?

https://github.com/sigp/lighthouse/blob/c71e011a1cc4482c2d31b0d37481e5b4b6dc2ac2/beacon_node/network/src/sync/block_lookups/tests.rs#L172-L177

Reply:
good spot!

Reply:
Fixing this revealed a bug! big ups! https://github.com/realbigsean/lighthouse/blob/refactor-deneb-networking/beacon_node/network/src/sync/network_context.rs#L446-L447